packages/block-editor/src/components/alignment-control/test/index.js

```diff
@@ -75,7 +75,7 @@ describe( 'AlignmentUI', () => {
 		const customControls = wrapperCustomControls.props().controls;
 		expect( customControls ).toHaveLength( 2 );
 
-		// should correctly call on change when right alignment is pressed (active alignment)
+		// Should correctly call on change when right alignment is pressed (active alignment)
```

Comment:
Seems that is not adding a dot after symbols > )

Reply:
See  https://github.com/WordPress/gutenberg/pull/37006#discussion_r759472277